### PR TITLE
Update using_letsencrypt.adoc

### PR DESCRIPTION
**Pull request by ChrisEdS**
Add missing `sh` to `sudo /etc/letsencrypt/` command

### CODE REVIEW

**Comment by voroyam** [comment]
because it was tested by @ChrisEdS I approve this PR

**Review by voroyam — APPROVED** [review]

**Comment by mmattel** [comment]
pls add a note for backporting and do post merging

File: modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc [hunk 1]
```diff
@@ -264,7 +264,7 @@ following command:
 
 [source,console]
 ----
-sudo /etc/letsencrypt/<your-domain-name>.sh
+sudo sh /etc/letsencrypt/<your-domain-name>.sh
```
Comment (phil-davis) [inline modules/admin_manual/pages/installation/letsencrypt/using_letsencrypt.adoc:267]: Why is this needed? I thought `sudo` would be fine running a shell script directly with an absolute path.
